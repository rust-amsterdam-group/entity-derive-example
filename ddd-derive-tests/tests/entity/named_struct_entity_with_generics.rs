use ddd_derive::Entity;

#[derive(Entity)]
struct Person<T> {
    raw: T,
    name: String,
}

fn main() {}