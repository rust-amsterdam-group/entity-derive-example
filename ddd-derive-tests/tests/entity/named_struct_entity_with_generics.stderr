error: Entity cannot be derived from Struct with Generic Type Parameters

  = note: Entity may only be derived from a Struct with named fields
  = help: Try changing the item to a Struct with named fields

 --> $DIR/named_struct_entity_with_generics.rs:4:1
  |
4 | / struct Person<T> {
5 | |     raw: T,
6 | |     name: String,
7 | | }
  | |_^
